sogs/__init__.py

```diff
@@ -3,8 +3,3 @@
 from . import config
 from . import logging
 from . import db
-from . import routes
```

Comment:
this file is best to be totally empty but still existing. it was only there to have it runnable via `sogs:app`